src/core/Store.ts

```diff
@@ -1,18 +1,19 @@
 import * as OrbitdbStore from "orbit-db-store";
 import * as OrbitDB from "orbit-db";
 import Collection from "./Collection";
+import { StoreOptions, CollectionOptions, OrbitDBOptions } from "./interfaces";
 const Index = require("./StoreIndex");
 const debug = require("debug")("aviondb:store");
-let orbitdb;
+let orbitdb: OrbitDB;
 
 class Store extends OrbitdbStore {
   _type: string;
-  _orbitdb: any;
-  openCollections: {};
+  _orbitdb: OrbitDB;
+  openCollections: any;
   _index: any;
   events: any;
   address: any;
-  constructor(ipfs, id, dbname: string, options: any = {}) {
+  constructor(ipfs: any, id: any, dbname: string, options: any = {}) {
```

Comment:
`ipfs` should be of type `IPFS` from the js-ipfs package right?

Reply:
Yes, but as we don't import the IPFS class within the `aviondb` module, I can't pass the class directly. We can use IPFS Type definitions (I'll have to look for them). Any other ideas on how we can add the IPFS type?

Reply:
@deanpress This seems to be a good way to add `IPFS` type: https://github.com/ceramicnetwork/js-ceramic/blob/9c805a12951b89cdf6f420788543c462e962c4f9/packages/3id-did-resolver/src/%40types/ipfs.d.ts

Reply:
@vasa-develop Agree - we can just include the interface. There's another one here with js-ipfs types that was made last month: https://github.com/Xmader/ipfs-types